client/src/components/Movie.jsx

```diff
@@ -1,34 +1,29 @@
 import React from 'react';
+import { Container, MovieInfo, RoundedText, Text, RatingContainer, MovieTitle, FullStars} from '../styling.jsx';
 
 const Movie = ({ movie }) => (
-  <div className="personnelMovie">
-    <div className="movie_info">
-      <div className="title">{movie.title}</div>
-      <div className="container">
-        <div id="release_date" className="infoplate_text">{movie.release_date}</div>
-        <div id="mpaa_rating" className="infoplate_text rounded">{movie.rating}</div>
-        <div id="runtime" className="infoplate_text rounded">{movie.runtime}</div>
-      </div>
-      <div className="rating_container">
-        <span className="star_rating_container">
-          <div className="star_rating_empty">
-            <img src="https://mapquiz.app/fec/infoplate/Star_empty.svg"></img>
-          </div>
-          <div className="star_rating_full" style={{ width: `${movie.vudu_rating * 17.75}px` }}>
-            <img src="https://mapquiz.app/fec/infoplate/Star_full.svg"></img>
-          </div>
-        </span>
-        <div className="rt_icon">
-          <img src={`${movie.rt_rating > 50 ? 'https://mapquiz.app/fec/infoplate/Tomato_certified_fresh.svg' : 'https://mapquiz.app/fec/infoplate/Tomato_rotten.svg'}`}></img>
-        </div>
-        <div className="rt_rating">{movie.rt_rating}%</div>
-      </div>
-      <div className="text">{`Rent/Buy from ${movie.price}`}</div>
-    </div>
-    <div className="movie_thumbnail">
-      <img src={movie.thumbnail_url}></img>
-    </div>
-  </div>
+  <Container>
+    <MovieInfo>
+      <MovieTitle>{movie.title}</MovieTitle>
+      <Container>
+        <Text>{movie.release_date}</Text>
+        <RoundedText>{movie.rating}</RoundedText>
+        <RoundedText>{movie.runtime}</RoundedText>
+      </Container>
+      <RatingContainer margin={true}>
+        <RatingContainer margin={false} hover={true}>
+          <img src="https://fec1-arwen.s3.amazonaws.com/infoplate/Star_empty.svg"></img>
+          <FullStars width={Number(movie.vudu_rating)}>
+            <img src="https://fec1-arwen.s3.amazonaws.com/infoplate/Star_full.svg"></img>
+          </FullStars>
+        </RatingContainer>
+        <img src={`${movie.rt_rating > 50 ? 'https://fec1-arwen.s3.amazonaws.com/infoplate/Tomato_certified_fresh.svg' : 'https://fec1-arwen.s3.amazonaws.com/infoplate/Tomato_rotten.svg'}`} style={{ display: 'flex', transform: 'scale(.8)' }}></img>
```

Comment:
are you planning on also showing if it's just normal fresh or if it's rotten?

Reply:
@HBMattoon I am using a ternary operator on line 20 to distinguish the two!